sympy/functions/special/tests/test_gamma_functions.py

```diff
@@ -442,3 +452,61 @@ def test_issue_14450():
 def test_issue_14528():
     k = Symbol('k', integer=True, nonpositive=True)
     assert isinstance(gamma(k), gamma)
+
+def test_multigamma():
+    from sympy import Product
+    p = Symbol('p')
+
+    assert str(multigamma(x, p)) == ('pi**(p*(p - 1)/4)*Product(gamma(-_k/2 + x + 1/2), '
```

Comment:
You can avoid string comparison by making use of `dummy_eq` (here and in other places).